coderedcms/fields.py

```diff
@@ -11,3 +12,16 @@ def __init__(self, *args, **kwargs):
     def formfield(self, **kwargs):
         kwargs['widget'] = ColorPickerWidget
         return super(ColorField, self).formfield(**kwargs)
+
+
+class ScriptField(models.TextField):
+    def __init__(self, *args, **kwargs):
+        super(ScriptField, self).__init__(*args, **kwargs)
+
+    def formfield(self, **kwargs):
+        kwargs['widget'] = Textarea(attrs={
+            'rows': 15,
+            'cols': 10,
+            'style': 'font-family:SFMono-Regular,Menlo,Monaco,Consolas,monospace;'
```

Comment:
I would prefer to use a CSS class rather than inline style... I think there is already a class named `monospace` (either in wagtail or coderedcms admin css). If not, we can add one.

Reply:
done. There is a class but I didn't to get specific for it to work.